bin/start_server.sh

```diff
@@ -12,9 +12,11 @@ cp -n /usr/src/app/ansible/roles/screenly/files/default_assets.yml /data/.screen
 
 echo "Running migration..."
 
-./manage.py initialize_assets
-./manage.py makemigrations
-./manage.py migrate --fake-initial
+# The following block ensures that the migration is transactional and that the
+# database is not left in an inconsistent state if the migration fails.
+cp /data/.screenly/screenly.db /data/.screenly/backup.db
+./manage.py migrate --fake-initial --database=backup
+cp /data/.screenly/backup.db /data/.screenly/screenly.db
```

Comment:
Hmm... this way, we lose the old database. 

```suggestion
cp /data/.screenly/screenly.db /data/.screenly/backup.db
cp /data/.screenly/screenly.db /data/.screenly/screenly.db.bak
./manage.py migrate --fake-initial --database=backup
mv /data/.screenly/backup.db /data/.screenly/screenly.db
```